spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonClientHttpRequestFactory.java

```diff
@@ -38,12 +34,8 @@
 
 	private final SpringClientFactory clientFactory;
 
-	private LoadBalancerClient loadBalancer;
```

Comment:
I'm trying to remember why we did that in the first place. I'm thinking it was to use the `LoadBalancerClient` abstraction, but at that point, we're in the middle of ribbon already.